src/pages/why.tsx

```diff
@@ -0,0 +1,412 @@
+                    {
+                        url: 'posthog-just-does-the-right-thing',
+                        value: 'PostHog just does the right thing',
+                        depth: 0,
+                    },
+                    {
+                        url: 'support-from-actually-technical-people',
+                        value: 'Support from actually technical people',
+                        depth: 0,
+                    },
+                    {
+                        url: 'posthog-is-more-popular-than-every-other-product',
+                        value: 'PostHog is more popular than every other product',
+                        depth: 0,
+                    },
+                    {
+                        url: 'who-doesnt-posthog-work-well-for',
+                        value: "Who doesn't PostHog work well for?",
+                        depth: 0,
+                    },
+                    {
+                        url: 'every-product-has-a-huge-free-tier',
+                        value: 'Every product has a huge free tier',
+                        depth: 0,
+                    },
+                ]}
+            >
+                <section className="mx-auto">
+                    <div className="flex flex-col md:items-center md:justify-end md:flex-row-reverse gap-8 md:gap-2">
+                        <div className="md:flex-1">
+                            <div className="@container article-content">
+                                <div className="opacity-70 text-[15px] font-medium">Why PostHog?</div>
+                                <h1 id="posthog-is-literally-designed-to-be-a-no-brainer" className="text-2xl">
+                                    PostHog is literally designed to be a no-brainer
+                                </h1>
+                                <p>
+                                    <strong>
+                                        PostHog is an entire suite of products you can use to make your software
+                                        successful – <em>far more than anyone else.</em>
+                                    </strong>
+                                </p>
+                                <p>
+                                    All that context in one spot so you can focus on your users and not your data
+                                    engineering to fix endless system integrations.
+                                </p>
+                                <p>
+                                    <strong>We have 10+ products today,</strong> but even if we don't have it, we will
+                                    eventually. We're going to build every piece of SaaS you need to make your product
+                                    successful. You read that right. We've already got the world's best track record at
+                                    delivering them by being the wisest with so many customers.
+                                </p>
```

Comment:
```suggestion
                                <p>
                                    <strong>We have 10+ products today,</strong> but even if we don't have it, we will
                                    eventually. 
                                </p>
                                <p>
                                    We're going to build every piece of SaaS you need to make your product
                                    successful. You read that right.
                                </p>     
                                <p>   
                                    We've already got the world's best track record at
                                    delivering them by being the widest with so many customers.
                                </p>
```

Splitting this up into shorter paras will make this more readable. I don't love the final line here, btw.  I assume that's meant to be "widest" rather than "wisest", but either way I think we could lose it.